src/main/java/frc/robot/subsystems/ArmSubsystem.java

```diff
@@ -23,6 +23,7 @@ public class ArmSubsystem extends SubsystemBase {
 
   public ArmSubsystem() {
     armMotor = new WPI_TalonFX(Constants.ArmConstants.ARM_MOTOR);
+    armMotor.setNeutralMode(NeutralMode.Brake);
```

Comment:
Not auto-balance related, so it should probably be in its own branch.

In general, there are many reasons why it's a good idea to keep separate work in separate branches.  In this case, maybe you'll end up using Jhony's branch for the final version of the auto-balancing code.  If that happens, you don't want to forget that you also want to add this line!